sunpy/sun/models.py

```diff
@@ -56,13 +56,12 @@
            10.157, 5.566, 2.259, 0.4483, 0.1528,
            0.042, 0.00361, 1.99e-7]
 
-_d = {'mass': _mass, 'luminosity': _luminosity, 'temperature': _temperature, 'density': _density}
-interior = pandas.DataFrame(_d, index = _radius)
-interior.index.name = 'radius'
+interior = pandas.DataFrame((pandas . _mass , pandas . _luminosity,pandas . _temperature, pandas . _density, pandas ._radius),(columns=['mass', 'luminosity', 'temperature','density' ,'radius'])
```

Comment:
interior = pandas.DataFrame((pandas . _mass , pandas . _luminosity,pandas . _temperature, pandas . _density, pandas ._radius),(columns=['mass', 'luminosity', 'temperature','density' ,'radius']))

Reply:
i forgot ')'
this code.